test/state_test.exs

```diff
@@ -59,6 +71,27 @@ defmodule State_Test do
                  {2, 1} => 8,
                  {2, 2} => 9
                },
+               player: "O"
+             }
+  end
+
+  test "sets player symbol \"O\" in board and switches player symbol to \"X\"" do
+    s = %State{player: "O"}
```

Comment:
There are a few exceptions to this, but those aside, we will almost always prefer descriptive variable names over single-character ones.  Since this is a test, I'd even say go a bit further than that, with the variable names.  This should be as easy to read as possible for a future developer, so like, maybe something like:

`empty_board`

...Or something like that.

Then line 82 would look like `assert State.set_move(move, empty_board)`.  If I see that in test output, or I hop directly to that line, then it gives me a good impression of what's going on _without looking around the rest of the file first_.  That's a huge boost in terms of cognitive help for new readers, or even yourself after you've slept a few nights and worked on other things.